jest-playwright.config.ts

```diff
@@ -0,0 +1,10 @@
+const config = {
+  browsers: ['chromium'],
+  collectCoverage: false,
```

Comment:
Should `collectCoverage` be true?

Reply:
I've used jest's own coverage collection configuration rather than `NYC`. We are just using `NYC` for merging and reporting.

Reply:
Oh, can `jest` collect **_browser_** coverage now?

In the past, when using `mocha`, I needed Istanbul/NYC to instrument the compiled [js] files _before_ sending them to the browser to collect coverage back. It wasn't possible otherwise.

I'm wondering why `jest-playwright` would expose an option to [collect coverage](https://github.com/playwright-community/jest-playwright#tracking-the-coverage) this way, via Istanbul instrumentation, if that was possible already with `jest` alone?

Reply:
Apologies at first look I believed this referred to `NYC`. So jest does have [Istanbul integrated](https://istanbul.js.org/docs/tutorials/jest/) for coverage collection / reporting hence not needing the additional configuration that mocha does, and there are references in our deps to `jest-babel` that would be required for the transforms, however I need to have a bit of a further dig as to why that is a requirement. As currently I am seeing no difference in our own reporting if that flag is flipped...